olm/addDigests.sh

```diff
@@ -59,9 +71,19 @@ do
   if [ "${names}" != " " ]; then
     RELATED_IMAGES="${RELATED_IMAGES},"
   fi
-  RELATED_IMAGES="${RELATED_IMAGES} { name: \"${name}\", image: \"${dest}\"}"
+  RELATED_IMAGES="${RELATED_IMAGES} { name: \"${name}\", image: \"${dest}\", tag: \"${source}\"}"
   names="${names} ${name} "
 done
 RELATED_IMAGES="${RELATED_IMAGES} ] } }"
 mv ${CSV_FILE} ${CSV_FILE}.old
 yq -Y "$RELATED_IMAGES" ${CSV_FILE}.old > ${CSV_FILE}
+sed -i ${CSV_FILE} -r -e "s|tag: |# tag: |" 
+rm -f ${CSV_FILE}.old
+
+# update original file with generated changes
```

Comment:
you may decide you're not ready for this step. But we're doing it in downstream now.

cc: @davidfestal @tolusha